SparkPost/tools-ui
# FE-1396: Address Dependabot alerts

- Addressed all non-low severity dependabot issues.
- Updated webpack to v3.

How to test:

1. First run `npm install`
2. Run `npm start` and verify that the local dev build works.
3. Run `num run build` and verify that production build is built correctly.
4. Run `serve -s build` and verify that the production build works.
